src/std/crypto/etc.ss

```diff
@@ -126,14 +126,12 @@
           buf)
         #f))))
 
-(def *urandom* (open-input-file "/dev/urandom"))
```

Comment:
This looses functionality i think; it was possible previously to seed the rng with something useful for reproducible runs (say a file read from /dev/urandom).
We no longer have this capability.

Reply:
Reproducibility is important for testing, and non-reproducibility for security. I think there should be two separate primitives for each, and code that needs to be tested with one yet run with the other might have a vector to choose between the two.

The crypto library looks like the right place for the actual non-reproducible bit. Akshully, we'd ideally add the additional functions for entropy control, too.